### PR TITLE
update events links

### PR DESCRIPTION
**Pull request by Arhell**
resove https://github.com/opensourcedesign/opensourcedesign.github.io/issues/401
all links checked

### CODE REVIEW

**Review by AnXh3L0 — DISMISSED** [review]

File: _events/2016-03-18-fossasia.md
Comment (Erioldoesdesign) [inline _events/2016-03-18-fossasia.md:13]: Is this correct? looks like a link was replaced with an 's' to me??

Comment (Erioldoesdesign) [reply _events/2016-03-18-fossasia.md:13]: @Arhell

**Review by Erioldoesdesign — CHANGES_REQUESTED** [review]
Other than the FOSS Asia one I don't see any errors 😄

File: _events/2016-03-18-fossasia.md
Comment (Arhell) [reply _events/2016-03-18-fossasia.md]: Done thanks

**Review by AnXh3L0 — APPROVED** [review]

**Review by SaptakS — APPROVED** [review]

**Comment by SaptakS** [comment]
@Erioldoesdesign this PR is waiting on your approval since you requested changes

**Review by Erioldoesdesign — APPROVED** [review]
LGTM

**Pull request merged**
